### PR TITLE
Feature image crop

### PR DESCRIPTION
**Pull request by bricesanchez**
* Use Cropper.js lib (https://github.com/fengyuanchen/cropperjs)
* No dependency to jQuery
* Add Ability to write multiple crops in Admin Image Edit form
* Use pre-defined or custom crop ratios
* Add Ability to use crop in WYSIWYG editors and in Image insert dialog

screencapture-localhost-3000-refinery-images-7-edit-2018-07-12-22_14_18

screencapture-localhost-3000-refinery-pages-test-edit-2018-07-12-22_15_00

screencapture-localhost-3000-refinery-pages-test-edit-2018-07-12-22_15_24

No tests added for this feature, sorry :(

### CODE REVIEW

**Comment by parndt** [comment]
That's pretty exciting @bricesanchez

**Comment by parndt** [comment]
keen for @anitagraham to take a look

File: images/app/views/refinery/admin/images/_form.html.erb [hunk 1]
```diff
@@ -108,4 +108,8 @@
     <%= javascript_include_tag 'refinery/plugins/cropper' %>
     <%= javascript_include_tag 'refinery/image_crop' %>
   <% end %>
+
+  <% content_for :stylesheets do %>
+    <%= stylesheet_link_tag('refinery/plugins/cropper') %>
+  <% end %>
```
Comment (parndt) [inline images/app/views/refinery/admin/images/_form.html.erb]: if it's just one, you can normally in-line it:

```erb
<% content_for :stylesheets, stylesheet_link_tag('refinery/plugins/cropper') -%>
```

**Comment by bricesanchez** [comment]
I've made some change to support a message if there is no crop. Could you review it please?

File: images/app/views/refinery/admin/images/_form.html.erb [hunk 2]
```diff
@@ -96,11 +96,15 @@
...
       <h2><%= t('.available_crops') %></h2>
 
-      <ul id="crop-list">
-        <% @image.crops.each do |crop| %>
-          <%= render partial: 'crop', locals: { crop: crop } %>
-        <% end %>
-      </ul>
+      <% if @image.crops.present? %>
+        <ul id="crop-list" class="clearfix">
+          <% @image.crops.each do |crop| %>
```
[the remark this answers is not in the document]
Comment (bricesanchez) [reply images/app/views/refinery/admin/images/_form.html.erb]: Yes it works :)

**Comment by bricesanchez** [comment]
Hi @anitagraham, do you have the time to check this PR?

**Comment by bricesanchez** [comment]
Could we merge it @parndt?

**Comment by parndt** [comment]
Yep 😄

**Comment by anitagraham** [comment]
This looks great. I am sorry I wasn't around to help review it - about to use it now, instead.

**Pull request merged**